juju/juju
# adding AssignProfiles to LXDProfileManager interface for instance mutater worker

## Description of change

Adding AssignProfiles to LXDProfileManager and LXDProfiler interfaces.  Also added lxd server UpdateContainerProfiles() for use by the implementations of AssignProfiles.  These are needed by the Instance Mutater worker.

## QA steps

Not hooked up for use yet.  No change  to juju functionality should currently be seen.
